spec/rails_helper.rb

```diff
@@ -40,7 +40,7 @@ def create_items
   end
 
   def create_user
-    let!(:user_one) { User.create(username: 'rossedfort', password: 'password') }
+    let!(:user_one) { User.create(username: 'rossedfort', password: 'password', first_name: 'Ross', last_name: 'Edfort', email_address: 'rossedfort@yahoo.com', phone_number: '908-698-9024') }
```

Comment:
Line is too long. [191/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.